lib/containers/container_images.pm

```diff
@@ -126,13 +126,14 @@ sub build_with_zypper_docker {
     zypper_call("in zypper-docker") if (script_run("which zypper-docker") != 0);
 
     my $log = "/tmp/zypper-docker-list-updates.log";
-    script_retry("zypper-docker list-updates $image >$log", timeout => 300, retry => 3, delay => 60);
+    my $zypp_dock = 'zypper-docker --gpg-auto-import-keys';
```

Comment:
Are you sure it is really necessary to include this key into every zypper call ? AFAIK if very first call will have it keys will be imported and all further calls won't ask any questions about keys anyway

Reply:
I'd rather use `zypper_docker` as variable name.

Reply:
That for sure works well when using zypper in a host system. Not sure whether docker or the container will save the state somehow. Maybe yes, but I find it as a safe action to include it in all calls